_inc/client/components/plans/plan-price/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { translate as __ } from 'i18n-calypso';
+import { getCurrencyObject } from '@automattic/format-currency';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export class PlanPrice extends Component {
+	render() {
+		const { currencyCode, rawPrice, original, discounted, className } = this.props;
+
+		if ( ! currencyCode || ! rawPrice ) {
+			return null;
+		}
+
+		// "Normalize" the input price or price range.
+		const rawPriceRange = Array.isArray( rawPrice ) ? rawPrice.slice( 0, 2 ) : [ rawPrice ];
+		if ( rawPriceRange.includes( 0 ) ) {
```

Comment:
Good catch! I've ported this code from Calypso but haven't considered some possibilities. Let's take care this in a subsequent PR or when we run into an issue with a `$0` price so that this PR is not blocked.